### ISSUE TITLE
Please do not delete the content

### ISSUE THREAD

**Issue by itjediFrank** [MicrosoftDocs/TeamBlogFeedback#176]
There are a lot of references to TechNet Gallery content in diverse blog posts (also from MS) or forum entries and so on. The migration for MSDN and TechNet blogs to https://techcommunity.microsoft.com/ or https://devblogs.microsoft.com/ was already a nightmare and is to some extent still a nightmare. Please do not repeat the same mistake!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 12b90746-323d-e43e-6dd4-97859717447e
* Version Independent ID: 5d68ad61-ec5a-b909-a8a9-65d5d2feb61e
* Content: [TechNet Gallery Retirement](https://docs.microsoft.com/en-us/teamblog/technet-gallery-retirement#feedback)
* Content Source: [teamblog/technet-gallery-retirement.md](https://github.com/MicrosoftDocs/blog/blob/live/teamblog/technet-gallery-retirement.md)
* Product: **team-blog**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by ChadSimmons** [comment]
There's so much good content in the gallery; I can't fathom Microsoft just deleting it after about a 3 month announcement.  With CodePlex, Microsoft had about a year of wind down and still hosts projects in an archive / read-only / download only, mode.  Surely the TechNet Gallery could get a similar level of treatment.

**Comment by sid590** [comment]
Great Article,loved the content.
Can the same be applied on www.bestbioforinstagram.in ?

**Comment by jpickerell** [comment]
Yes Microsoft, there is just too many good resources here for you to just flush down the toilet! Years and years of useful scripts in so many different areas! It might be an extremely low use/low traffic site, but when you need a good example, there are just too many here to lose access to!

Please reconsider!

**Comment by RufusVS** [comment]
I just found out about this archive.  I'd hate to lose it so soon!

**Comment by s31064** [comment]
Agreed.  Removing this repository would be a major mistake.  I don't care if it has to become read-only (which I think is also a mistake), but it needs to stay.  One centralized repository is far better than trying to search across God only knows how many places in GitHub, especially when you don't even know that the author migrated the content.

Please keep the gallery intact.
